pkgs/development/python-modules/sv-ttk/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, pkgs
+, python3
+, fetchPypi
+}:
+python3.pkgs.buildPythonPackage rec {
+  pname = "sv-ttk";
+  version = "2.4.5";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "sv_ttk";
+    hash = "sha256-ysRhRxrml+wmluH8F5AE7vZYXrTNUg5ZzI+26jwpOpc=";
+  };
+
+  meta = with lib; {
+    description = "A gorgeous theme for Tkinter/ttk, based on the Sun Valley visual style";
+    homepage = "https://github.com/rdbende/Sun-Valley-ttk-theme";
```

Tools (e.g., `nix-update`) can add additional details to the commit message if `changelog` is present.

```suggestion
    homepage = "https://github.com/rdbende/Sun-Valley-ttk-theme";
    changelog = "https://github.com/rdbende/Sun-Valley-ttk-theme/releases/tag/v${version}";
```

---

Nice! TIL.